starter-core/src/main/java/io/micronaut/starter/feature/CommunityFeature.java

```diff
@@ -48,6 +52,12 @@ default boolean isCommunity() {
 
     @Override
     default boolean isVisible() {
-        return CommunityFeatureValidator.ENABLE_COMMUNITY_FEATURES;
+        return supportsCurrentMicronautVersion();
+    }
+
+    default boolean supportsCurrentMicronautVersion() {
+        return VersionInfo.getMicronautMajorVersion()
+                .filter(integer -> builtWithMicronautVersion().getMajor() == integer)
```

Comment:
```suggestion
                .filter(majorVersion -> builtWithMicronautVersion().getMajor() == majorVersion)
```

variable called `integer` had me confused for at least 2 minutes 😉

Reply:
fixed via https://github.com/micronaut-projects/micronaut-starter/pull/2180/commits/2dd1dd4ac1bd25a17faabb4b448fd494641d5664